pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, fetchFromGitHub, glibcLocales
+, cmake, eigen, openmpi, gtest, python3
+}:
+
+stdenv.mkDerivation rec {
+  pname = "onnxruntime";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "microsoft";
+    repo = "onnxruntime";
+    rev = "v${version}";
+    sha256 = "0s8ylc5xr55490hbz7zn3hnp9dnyp92d320ln8xw5hqkw3mgyr3p";
+    # TODO: use nix-versions of grpc, onnx, eigen, googletest, etc.
+    # submodules increase src size and compile times significantly
+    # not currently feasible due to how integrated cmake build is with git
+    fetchSubmodules = true;
+  };
+
+  # TODO: build server, and move .so's to lib output
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [
+    cmake
+    python3 # for shared-lib or server
+  ];
+
+  cmakeDir = "../cmake";
+
+  cmakeFlags = [
+    "-Donnxruntime_USE_OPENMP=ON"
+    "-Donnxruntime_BUILD_SHARED_LIB=ON"
+    "-Donnxruntime_USE_LTO=ON"
+  ];
+
+  # ContribOpTest.StringNormalizerTest sets locale to en_US.UTF-8"
+  preCheck = ''
+    export LOCALE_ARCHIVE="${glibcLocales}/lib/locale/locale-archive"
+  '';
+  doCheck = true;
+
+  postInstall = ''
+    rm -r $out/bin   # ctest runner
+    rm -r $dev/lib/  # protobuf artifacts, avoid false duplicates
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Cross-platform, high performance scoring engine for ML models ";
+    longDescription = ''
+        ONNX Runtime is a performance-focused complete scoring engine
```

Comment:
Genuine question: is it ok for `longDescription` to start with a tab/spaces?

Reply:
I'm not actually sure about this...

Reply:
oh wait, I am sure because of how the "lines" type in nixpkgs works... all preceding whitespace accross each line gets truncated. https://nixos.wiki/wiki/Nix_Expression_Language#Types